metadata-ingestion/src/datahub/ingestion/source_config/sql/snowflake.py

```diff
@@ -105,15 +105,21 @@ class BaseSnowflakeConfig(BaseTimeWindowConfig):
     password: Optional[pydantic.SecretStr] = pydantic.Field(
         default=None, exclude=True, description="Snowflake password."
     )
+    private_key: Optional[str] = pydantic.Field(
+        default=None,
+        description="Private key in a form of '-----BEGIN PRIVATE KEY-----\\nprivate-key\\n-----END PRIVATE KEY-----\\n' if using key pair authentication. Encrypted version of private key will be in a form of '-----BEGIN ENCRYPTED PRIVATE KEY-----\\nencrypted-private-key\\n-----END ECNCRYPTED PRIVATE KEY-----\\n' See: https://docs.snowflake.com/en/user-guide/key-pair-auth.html",
```

Comment:
Do we need the \\n shouldn't \n enough in the string?

Reply:
Check out how this looks like in bigquery -> https://datahubproject.io/docs/generated/ingestion/sources/bigquery/#module-bigquery